bot/cogs/puzzles.py

```diff
@@ -800,7 +803,7 @@ async def archive_solved(self, ctx, *, minutes: Optional[int]):
         logger.info(message)
         await ctx.send(message)
 
-    @tasks.loop(seconds=30.0)
+    @tasks.loop(seconds=2.0)
```

Comment:
This was a mistake and will be reverted on the next commit (I dropped it down for testing)